app/components/CategorySection.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { View, ScrollView, StyleSheet, Text, TouchableOpacity } from "react-native";
+import { colors, getStyles } from "../constants/styles/global";
+
+const CategorySection = () => {
+    const { theme } = getStyles();
+
+    /* this will later be extracted from db maybe? */
+    const categories = [
+        "High-Protein",
+        "Low-Calorie",
+        "Healthy",
+        "Vegan",
+        "Vegitarian",
+        "Halal",
+    ];
```

Comment:
You'll be able to find the following under `food.nutrionalData`:

```
  isGlutenFree: boolean;
  isDairyFree: boolean;
  isVegetarian: boolean;
  isVegan: boolean;
  isEstimate: boolean;
  hasFreshFruit: boolean;
  hasFreshVegetables: boolean;
  ```

Reply:
Sounds good! I just had those there temporarily until we got the firebase stuff setup